src/normal.c

```diff
@@ -4074,6 +4080,12 @@ nv_gotofile(cmdarg_T *cap)
 	return;
 #endif
 
+    if (curwin->w_p_wfb)
+    {
+        semsg("%s", e_winfixbuf_cannot_go_to_buffer);
+        return;
```

Comment:
and here please

Reply:
These tests currently use this code (e.g. if you remove this block, the following tests immediately fail)

Test_normal_gF()
Test_normal_gf()
Test_normal_square_bracket_left_f()
Test_normal_square_bracket_right_f()

Reply:
okay thanks